src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private Color color = ColorSupplier.getRandomColor();
+    private int number = new Random().nextInt(100);
```

Comment:
This code must be in the Lottery class. 
He is responsible for creating Ball objects.